vispy/visuals/volume.py

```diff
@@ -698,4 +711,4 @@ def draw(self, transforms):
             self._program['u_threshold'] = self._threshold
         
         # Draw!
-        self._program.draw('triangles', self._index_buffer)
+        self._program.draw('triangles', self._index_buffer)
```

Comment:
Travis will be mad that this doesn't have a trailing newline, consider using `make flake` to check for other probs